### ISSUE TITLE
Typo in chunk(self, seek=None, lenient=False): Unknown variable

### ISSUE THREAD

**Issue by heuer** [drj11/pypng#48]
See: <https://github.com/drj11/pypng/blob/master/code/png.py#L1437>
```python
            if len(checksum) != 4:
                raise ValueError('Chunk %s too short for checksum.', tag)
```
should probably changed to:
```python
            if len(checksum) != 4:
                raise ValueError('Chunk %s too short for checksum.' % type)
```

**Comment by heuer** [comment]
Arrg. Sorry, already covered by pull request #44

**Comment by heuer** [comment]
Fixed in commit 12173cd
